pkg/sql/opt/ops/statement.opt

```diff
@@ -357,3 +357,10 @@ define AlterRangeRelocatePrivate {
     # Props stores the required physical properties for the input expression.
     Props PhysProps
 }
+
+# DeleteBatch represents a `DELETE BATCH FROM .. ` statement.
+[Relational]
+define DeleteBatch {
+    # Delete is the original delete statement.
+    Delete Delete
```

Comment:
To match other expressions that contain AST nodes:

```suggestion
    Syntax Delete
```

Reply:
Done